_layouts/robot.html

```diff
@@ -56,5 +58,56 @@
     </div>
     {% endif %}
 
-</div>
+    <div class="column is-6">
+        {% if page.Specifications %}
+        <div class="content">
+            <h2>Specifications</h2>
+            <ul>
+                {% for Specification in page.Specifications %}
+                <li>{{ Specification.item }}</li>
+                {% endfor %}
+            </ul>
+        </div>
+        {% endif %}
+        {% if page.Capabilities %}
+        <div class="content">
+            <h2>Capabilities</h2>
+            <ul>
+                {% for Capability in page.Capabilities %}
+                <li>{{ Capability.item }}</li>
+                {% endfor %}
+            </ul>
+        </div>
+        {% endif %}
+    </div>
 
+    <div class="column is-6">
+
+
+        {% if page.Competitions %}
+        <div class="content">
+            <h2>Competitions</h2>
+            <ul>
+                {% for Competition in page.Competitions %}
+                <li>{{ Competition.item }}</li>
+                {% endfor %}
+            </ul>
+        </div>
+        {% endif %}
+        {% if page.Capabilities %}
+        <div class="content">
+            <h2>Links</h2>
```

Comment:
Do no show the "Links" section if no links exist.

Reply:
https://github.com/FRCTeam1987/teambroncobots.com/pull/50/commits/f23594b3fcb32a5f7637a99288e87e3d1dd50c12